deploy-cluster-aws/fabfile.py

```diff
@@ -138,13 +133,26 @@ def configure_bigchaindb():
     run('bigchaindb -y configure', pty=False)
 
 
+# Send the specified configuration file to
+# the remote host and save it there in
+# ~/.bigchaindb
+# Use in conjunction with set_host()
+# No @parallel
+@task
+def send_confile(confile):
+    put('confiles/' + confile, 'tempfile')
+    sudo('mv tempfile ~/.bigchaindb')
```

Comment:
Good question. Originally, I tried to do something like `put('confiles/' + confile, '~/.bigchaindb')` but I was getting an error about not having sufficient permissions. So I just did it in two steps, using sudo to move the file into ~.

I was curious to see if sudo() was necessary, using run() instead. It works with run()!

I'll push the new code that uses run() instead.